### ISSUE TITLE
feature request: Gaussian Mixture Models as a cluster method

### ISSUE THREAD

**Issue by jeremymanning** [ContextLab/hypertools#143]
this could provide a better clustering solution, e.g. for the [mushrooms clustering example](https://github.com/ContextLab/hypertools-paper-notebooks/blob/master/mushrooms.ipynb)

sklearn has a nice implementation that we could wrap: http://scikit-learn.org/stable/modules/generated/sklearn.mixture.GaussianMixture.html#sklearn.mixture.GaussianMixture

**Comment by jeremymanning** [comment]
Addressed in the HyperTools 1.0 update (PR #272, https://github.com/ContextLab/hypertools/pull/272): `hyp.cluster()` now supports `GaussianMixture` (and `BayesianGaussianMixture`) directly, returning soft per-observation membership proportions instead of hard labels.

```python
import hypertools as hyp
import numpy as np
data = np.random.randn(60, 5)
props = hyp.cluster(data, cluster='GaussianMixture', n_clusters=3)
print(props.shape)   # (60, 3) -- rows sum to 1
```

`n_clusters` maps to `n_components` under the hood, so the API stays consistent with the other cluster methods, and `hyp.plot(data, cluster='GaussianMixture', n_clusters=3)` works end-to-end (verified visually).

Closing as addressed in the 1.0 update — please reopen if you hit anything else.
